### PR TITLE
Manager README

### PR DESCRIPTION
**Pull request by dmleong**
👉  [Rendered](https://github.com/dmleong/manager-resources/blob/a27d9a9bfb56272cc325e281a597389bf9a5954c/README.md) 👈 

This is the first draft of my philosophy as a manager, the best way to work with me, my values, and what I expect from those who work for me. 

Also covers feedback and 1:1s

Feedback welcome!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,83 @@
...
+
+[Radical Candor](https://www.blinkist.com/en/books/radical-candor-en/) is perhaps the best resource to understand my approach to management and feedback. I believe in giving timely, honest feedback _because I care about you._ I don't believe in surprising people at review time with bad feedback. Your performance reviews should be underwhelming events because you should always know where you stand with me at all times. You can read more about my thoughts on feedback [here](https://medium.com/@tsunamino/setting-goals-with-your-engineers-that-dont-completely-suck-cb76b87e4275).
+
+## My role
+
+I like to think of myself as a momma bear coaching a soccer team that my cubs are on. I will 100% eat any other bears that dares to hurt my team, but I also expect my cubsters to be on their best behavior and to be excellent team players. Everyone must shake hands with the other team at the end of the game. No exceptions.
```
Comment (gallexi) [inline README.md]: ...should it be "everyone must shake paws with the other team"?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,83 @@
...
+- get you the resources you need
+- challenge you in ways that will help you grow
+- champion your accomplishments to anyone who will listen (and even those who won't)
+
+My role is not to:
+
+- do your job for you
+- enable laziness
+
+You do have to meet me halfway. I will be your biggest cheerleader, but I can't be everywhere at once! Sometimes I'll ask you to write down what your doing to help me out. Sometimes it's because I legit have too many things to focus on and I need an easy way to recall all the amazing things that you've done lately. Help me help you get more money!
```
Comment (gallexi) [inline README.md]: "what your doing" => "what you're doing"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,83 @@
...
+
+I expect you to show up on time and be present at meetings. I hate useless meetings and promise to keep them to a minimum. There will always be an agenda. They'll probably always be remote meetings. So put on some pj pants and hop on video and be fully present for the short time we have together.
+
+I expect you to help out your fellow teammates if they need help. We all have trouble here and there and no one likes to feel like a burden! If someone is asking for help then let's help teach one another the skills we need to solve the problem.
+
+I expect you to be respectful of one another's lived experiences. I hate to use a buzzword, but diversity & inclusion is super important to making good teams. That means sometimes misunderstandings happen! But I expect you to be respectful of someone's lived experiences. However, keep in mind that I value the safety of the marginalized over the hurt feelings of those with more privilege. Power dynamics in terms of race, gender and gender identity, sexual identity or expression, class, disability, religion, age, body size, etc will always be kept into consideration. Clarifying questions: good. Dismissing someone's lived experiences: bad.
```
Comment (gallexi) [inline README.md]: > gender and gender identity

Not super sure what you mean here? Possibly "gender identity and expression"?

Comment (gallexi) [reply README.md]: > kept into consideration

"taken into" or "kept in", I think. Unsure though 🤔

**Review by gallexi — APPROVED** [review]
I love this it is very awesome just a few lil grammar nits and a question and one probably annoying comment about bear hands vs paws 😜

File: README.md (hunk 2 above)
Comment (dmleong) [reply README.md]: Yikes. I must've been real tired

File: README.md (hunk 3 above)
Comment (dmleong) [reply README.md]: > Possibly "gender identity and expression"?

Yes, that's what I meant! Should've copied and pasted from a better source than my memory

**Comment by dmleong** [comment]
@gallexi I believe I've addressed your feedback! Can I get another 👀please?

**Comment by gallexi** [comment]
Looks good!

**Comment by dmleong** [comment]
Gonna merge this in for now. Updates and suggestions are always welcome 🌈

**Pull request merged**